CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ ___Note:__ Yet to be released changes appear here._
 * `DEPS`: update to `diagram-js@12.8.1`
 * `DEPS`: update to `dmn-js@14.7.1`
 * `DEPS`: update to `dmn-js-properties-panel@3.2.1`
+* `DEPS`: update to `@bpmn-io/form-js@1.6.0`
```

Comment:
I'll move this, as it did not land in 5.18 but next.

Reply:
Done via https://github.com/camunda/camunda-modeler/pull/4026/commits/83b1ce5af6ee6578fd2dc385e9fe1634845aaebb.